src/test/java/uk/ac/ebi/eva/pipeline/io/writers/StatisticsMongoWriterTest.java

```diff
@@ -0,0 +1,103 @@
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import uk.ac.ebi.eva.pipeline.configuration.CommonConfiguration;
+import uk.ac.ebi.eva.pipeline.configuration.JobOptions;
+import uk.ac.ebi.eva.pipeline.model.PopulationStats;
+import uk.ac.ebi.eva.test.data.VariantData;
+import uk.ac.ebi.eva.test.utils.JobTestUtils;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+import static junit.framework.TestCase.assertTrue;
+
+/**
+ * {@link StatisticsMongoWriter}
+ * input: a List of {@link PopulationStats} to each call of `.write()`
+ * output: the FeatureCoordinates get written in mongo, with at least: chromosome, start and end.
+ */
+@RunWith(SpringJUnit4ClassRunner.class)
+@ContextConfiguration(classes = { JobOptions.class, CommonConfiguration.class})
+public class StatisticsMongoWriterTest {
+
+    @Autowired
+    public JobOptions jobOptions;
+
+    @Test
+    public void shouldWriteAllFieldsIntoMongoDb() throws Exception {
+        String dbName = jobOptions.getDbName();
+        JobTestUtils.cleanDBs(dbName);
+
+        String statsPath = VariantData.getPopulationStatsPath();
+        JsonLineMapper mapper = new JsonLineMapper();
+        Map<String, Object> map = mapper.mapLine(statsPath, 0);
+        PopulationStats populationStats = new PopulationStats();
+        populationStats.setCohortId((String)map.get("cid"));
+        populationStats.setVariantId((String)map.get("vid"));
+        populationStats.setStudyId((String)map.get("sid"));
+        populationStats.setMaf((Double)map.get("maf"));
+        populationStats.setMgf((Double)map.get("mgf"));
+        populationStats.setMafAllele((String)map.get("mafAl"));
+        populationStats.setMgfGenotype((String)map.get("mgfGt"));
+        populationStats.setMissingAlleles((Integer) map.get("missAl"));
+        populationStats.setMissingGenotypes((Integer) map.get("missGt"));
+        populationStats.setGenotypeCount((Map<String, Integer>) map.get("numGt"));
+
+
+        List<PopulationStats> populationStatsList = Arrays.asList(populationStats);
+
+        // do the actual writing
+        StatisticsMongoWriter statisticsMongoWriter = new StatisticsMongoWriter(
+                jobOptions.getMongoOperations(), jobOptions.getDbCollectionsStatsName());
+
+        statisticsMongoWriter.write(populationStatsList);
+
+
+        // do the checks
+        MongoClient mongoClient = new MongoClient();
+        DBCollection statsCollection = mongoClient.getDB(dbName).getCollection(jobOptions.getDbCollectionsStatsName());
+
+        // count documents in DB and check they have region (chr + start + end)
+        DBCursor cursor = statsCollection.find();
+
+        int count = 0;
+        while (cursor.hasNext()) {
+            count++;
+            DBObject next = cursor.next();
+            assertTrue(next.get("cid") != null);
```

Comment:
`assertNotNull` is available. Also please use asserts from package `org.junit.Assert`, because `junit.framework` is deprecated.